packages/components/src/spacer/README.md

```diff
@@ -0,0 +1,112 @@
+# Spacer
+
+`Spacer` is a primitive layout component that providers inner (`padding`) or outer (`margin`) space in-between components. It can also be used to adaptively provide space within an `HStack` or `VStack`.
+
+## Table of contents
+
+## Usage
+
+`Spacer` comes with a bunch of shorthand props to adjust `margin` and `padding`. The values of these props work as a multiplier to the library's grid system (base of `4px`).
+
+```jsx
+import { Spacer } from '@wordpress/components';
+
+function Example() {
+	return (
+		<View>
+			<Spacer>
+				<Heading>WordPress.org</Heading>
+			</Spacer>
+			<Text>
+				Code is Poetry
+			</Text>
+		</View>
+	);
+}
+```
+
+## Props
+
+##### m
```

Comment:
The names of all props here don't follow WordPress Coding Standards:

https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#naming-conventions

> Variable and function names should be full words, using camel case with a lowercase first letter. This is an area where this standard differs from the WordPress PHP coding standards.
> 
> Names should be descriptive, but not excessively so. Exceptions are allowed for iterators, such as the use of i to represent the index in a loop.

Is there any strong argument towards going with all those short names?